domain/entities/transfers/transfers.go

```diff
@@ -0,0 +1,37 @@
+package transfers
+
+import (
+	"errors"
+	"time"
+)
+
+var (
+	ErrInvalidLengthID = errors.New("empty ID not allowed")
+	ErrInvalidAmount   = errors.New("invalid amount, be must > 0")
+	ErrEqualAccounts   = errors.New("equal source and destination accounts are not allowed.")
+)
+
+type Transfer struct {
+	ID                   string    `json:"id"`
+	AccountOriginID      string    `json:"account_origin_id"`
+	AccountDestinationID string    `json:"account_destination_id"`
+	Amount               uint      `json:"amount"`
+	Created_At           time.Time `json:"created_at"`
```

Comment:
```suggestion
	CreatedAt           time.Time `json:"created_at"`
```
Acho que faltou aqui